lms/envs/common.py

```diff
@@ -2283,6 +2283,7 @@
 ################# Student Verification #################
 VERIFY_STUDENT = {
     "DAYS_GOOD_FOR": 365,  # How many days is a verficiation good for?
+    "VERIFICATION_EXPIRATION_DAYS": 28,  # How many days before we show message to reverify?
```

Comment:
This variable name and the accompanying comment can be clearer. They're vague right now. Clarity around this part of the system is valuable.

The variable represents the window within which a verification is considered to be "expiring soon." (This would make a better comment.)`EXPIRING_SOON_WINDOW` would be a more meaningful variable name. Since this is part of the verification settings dict, you don't need the word "verification" in there.